deploy/deploy_gocd.yml

```diff
@@ -5,7 +5,7 @@
   roles:
     - {role: common, when: "install_glusterfs"}
     - {role: puppet_disable, when: "disable_puppet"}
-    - {role: iptables, when: "iptables_config"}
```

Comment:
Oops, it occurred to me that we do in fact need this variable, as it controls whether or not to run the iptables commands in Dev/Test/Prod, where we usually do not want to.  Perhaps keep this as it was, and use a new variable inside the role instead of using the same variable?  Then leave iptables_config as True, and set that new one to False in Travis.